main.py

```diff
@@ -55,15 +54,24 @@
 emptyID = config["statics"]["emojisIDs"]["empty"]
 debugMode = config["debugMode"]
 
-# Code for debug
-from loguru import logger
+# Get values for database
+host = config["database"]["host"]
+user = config["database"]["user"]
+passwd = config["database"]["passwd"]
+database = config["database"]["database"]
```

Comment:
Separate database connection and configuration logic to another file and then import most necessary code inside main function.